src/app.js

```diff
@@ -34,19 +36,23 @@ class App {
   async listen () {
     await this.errors.init(this.app)
 
+    this.logger.info('creating app')
     const server = this.app
       .use(this.parser)
       .use(this.views)
       .use(this.router.routes())
       .use(this.router.allowedMethods())
       .listen(process.env.PORT || 7770)
+    this.logger.info('listening on :' + (process.env.PORT || 7770))
 
+    this.logger.info('attaching endpoints and connecting to moneyd...')
     await this.pubsub.init(server)
     await this.index.init(this.router)
     await this.send.init(this.router)
     await this.ping.init(this.router)
     await this.graph.init(this.router)
     await this.receiver.init(this.router)
+    this.logger.info('moneyd-gui ready (republic attitude)')
```

Comment:
"republic attitude", really? 😄

Reply:
what other kind of attitudes are there, even?